### PR TITLE
HTML/CSS/IMAGES for mobile home page of Portifolio Clone

### PR DESCRIPTION
**Pull request by nelsinofrancisco**
# Implementation of recent works and About-me Sections 

## Features Added:

- Recent works Section;
- Grid Layout in the main container (class="recent-works-container');
- Cards implementing a preview of recent works;
- About-me Section;
- A list of Languages/Frameworks/Skills owned;

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -27,6 +26,36 @@ body {
...
+.project-title-custom-font {
+  color: #3a4a42;
+  font-family: Inter, Georgia, 'Times New Roman', Times, serif;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: center;
+  margin: 0;
```
Comment (voscarmv) [inline css/styles.css]: - [ ] Please avoid excessive redundancy of CSS attributes. In this case, the `margin: 0` attributes appears 10 times in your style. Please use a single class for this attribute and use that instead. You may perform your own redundancy tests with this tool: https://courses.cs.washington.edu/courses/cse154/20su/resources/assets/css-redundancy-checker/index.html

**Review by voscarmv — CHANGES_REQUESTED** [review]
Hi @nelsinofrancisco , 

Good job so far!

- I really like how your project hits all the marks with regards to code. Well done.

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- Although your code is great, there is still an unfulfilled mandatory style requirement:
image

- Your design and the Figma guideline do not match with regards to color and font. Please fix this: 

| Your design | Figma design |
| --- | --- |
| image | image |

- Please also take care of the CSS best practices suggestions I left on my code comments.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding a screenshot to the README file.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 1 above)
Comment (o-rashad) [reply css/styles.css]: Please Follow the previous reviewer recommendation by using the referenced tool 👌🏼 
As it appears for _instance_ you used `display: flex;` for **15** times

_Suggistion_
- Create a class for example `dis-flex` with property `display: flex;` and append it to your desired elements in your HTML. 🤔

File: index.html [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+  <section class="recent-works-container">
+    <div class="recent-works intro-header">
+      <div class="intro-header primary-text">
+        <h2 class="primary-text first-color primary-text-custom-font">My Recent Works</h2>
+      </div>
+      <div class="intro-header indicator">  
+      </div>
+    </div>
+    <div class="card-work-container">
+      <div class="card-work card-1">
```
Comment (o-rashad) [inline index.html]: Example for the _[OPTIONAL CHANGE]_

That div and its peers could be replaced with an `<article>` tag.

**Review by o-rashad — CHANGES_REQUESTED** [review]

**Review by o-rashad — CHANGES_REQUESTED** [review]
Hi @nelsinofrancisco , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check my comments on your code._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @nelsinofrancisco , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**